microsoft-edge/web-platform/site-impacting-changes.md

```diff
@@ -37,6 +37,7 @@ This table lists high-impact changes which the Microsoft Edge team is tracking c
 | Insecure downloads over HTTP | Future release (TBD) | When a user tries to download potentially dangerous content from an HTTP site, the user will receive a UI warning, such as "Insecure download blocked."  The user will still have an option to proceed and download the item.  Admins can use the `InsecureContentAllowedForUrls` policy to specify HTTP sites for which the warning will be suppressed.  Admins can use the `InsecureDownloadWarnings` feature flag to test the impact of this upcoming feature. |
 | Deprecate unload event | Future release (TBD) | Introduces a new Permission-Policy to allow creating unload event listeners. The default policy is `allow`, but the default policy will gradually be migrated to `deny`, such that unload handlers stop firing on pages, unless a page explicitly opts in to re-enable them.  This change is happening in the Chromium project, on which Microsoft Edge is based.  For more information, see [Intent to Deprecate: Deprecate unload event](https://groups.google.com/a/chromium.org/g/blink-dev/c/dvusqw9-IhI/m/SBkm_u1RAQAJ). |
 | Deprecate `getHostEnvironmentValue` | v137-v141 | Deprecate the non-standards-based, Microsoft Edge-only `window.external.getHostEnvironmentValue()` method, in favor of using the standardized User-Agent Client Hints API to expose browser and platform information.  User-Agent Client Hints provide browser and platform information in a more privacy-preserving way.  See [Detect Windows 11 and CPU architecture using User-Agent Client Hints](./how-to-detect-win11.md), and [User-Agent Client Hints API](https://developer.mozilla.org/docs/Web/API/User-Agent_Client_Hints_API). |
+| HTTPS policy updates | v136-v139 | Deprecate the `AutomaticHttpsDefault` policy and replace it with the `HttpsUpgradesEnabled` <!-- link missing --> policy. The `AutomaticHttpsDefault` is deprecated in Microsoft Edge version 136 and is planned to be removed in Microsoft Edge version 139. The new `HttpsUpgradesEnabled` policy should be used instead, and is available in Microsoft Edge version 136. |
```

Comment:
try to avoid 'should'
https://learn.microsoft.com/en-us/style-guide/a-z-word-list-term-collections/s/should-vs-must
@captainbrosset